.github/workflows/main.yaml

```diff
@@ -8,7 +8,7 @@ on:
 name: Main
 jobs:
   Test:
-    name: 'Test'
+    name: 'Lint & test'
```

Comment:
I think it would be better to separate it into 2 different jobs.

Reply:
I hesitated. Ok. I do it.